usaspending_api/etl/elasticsearch_loader_helpers/utilities.py

```diff
@@ -127,7 +147,7 @@ def execute_sql_statement(cmd: str, results: bool = False, verbose: bool = False
 def db_rows_to_dict(cursor: psycopg2.extensions.cursor) -> List[dict]:
     """Return a dictionary of all row results from a database connection cursor"""
     columns = [col[0] for col in cursor.description]
-    return [dict(zip(columns, row)) for row in cursor.fetchall()]
+    return [dict(zip(columns, row, strict=False)) for row in cursor.fetchall()]
```

Comment:
Won't block over reverting this but `strict=False` is the default behavior for `zip()`.

Reply:
I'll remove this, because I don't remember changing this. 👻